chapters/6-analysis.tex

```diff
@@ -682,6 +714,24 @@ \subsection{Dynamic exploratory analysis}
 However, as with other Markdown options, the need to learn a new syntax may
 discourage take up among team members who don't work with GitHub more extensively.
 
+% Third best
+Whatever software you are using,
+what matters is that you make sure to implement a self-updating process for table and figures.
+We have given a few recommendations as to what we believe to be best practices,
+but you will need to find out what works for your team.
+If your team has decided to use Microsoft Office, for example,
+there are still a few options to avoid problems with copy-pasting.
+The easiest solution may be for the less code-savvy members of the team
+to develop the text of the final output pointing to exhibits that are not included inline.
+If all figures and tables are presented at the end of the file,
+whoever is developing the code can export them into a Word document using Markdown,
+so at least this file can be updated when the code changes.
+Finally, you can export tables to csv files,
```

Comment:
"CSV" or "`csv`"? -- formatting

Reply:
@bbdaniels to rephrase this and indicate that `putexcel` and `putword` are better solutions than exporting a csv file, because, as mentioned in chapter 2, absolute file pahts are buggy unless you can make them dyamic.